### PR TITLE
[CI] used vcpkg robotology prebuilt deps for Windows runners

### PR DESCRIPTION
**Pull request by pattacini**
This PR introduces the use of vcpkg robotology prebuilt packages for doing CI on Windows runners.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -51,13 +50,17 @@ jobs:
...
+        # To avoid spending a huge time compiling vcpkg dependencies, we download a root that comes precompiled with all the ports that we need 
+        choco install -y wget unzip
+        # To avoid problems with non-relocatable packages, we unzip the archive exactly in the same C:/robotology/vcpkg 
+        # that has been used to create the pre-compiled archive
+        cd C:/
+        md C:/robotology
+        md C:/robotology/vcpkg
+        wget https://github.com/robotology/robotology-superbuild-dependencies-vcpkg/releases/download/${env:vcpkg_robotology_TAG}/vcpkg-robotology.zip
+        unzip vcpkg-robotology.zip -d C:/robotology/vcpkg
+        # ${VCPKG_INSTALLATION_ROOT}/vcpkg.exe install --triplet x64-windows opencv3[contrib] vtk hdf5 fftw3
```
Comment (pattacini) [inline .github/workflows/ci.yml]: @traversaro if I enable this line, I'll get a CI error.
See https://github.com/robotology/assistive-rehab/runs/531463623?check_suite_focus=true

Comment (traversaro) [reply .github/workflows/ci.yml]: By default on Windows the shell is powershell, so you need to use env variables as `${env:VCPKG_INSTALLATION_ROOT}`

Comment (pattacini) [reply .github/workflows/ci.yml]: Thanks!

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -49,16 +41,26 @@ jobs:
...
+        md C:/robotology/vcpkg
+        wget https://github.com/robotology/robotology-superbuild-dependencies-vcpkg/releases/download/${env:vcpkg_robotology_TAG}/vcpkg-robotology.zip
+        unzip vcpkg-robotology.zip -d C:/robotology/vcpkg
+        # Overwrite the VCPKG_INSTALLATION_ROOT env variable defined by GitHub Actions to point to our vcpkg
+        echo "::set-env name=VCPKG_INSTALLATION_ROOT::C:/robotology/vcpkg"
         
+    - name: Dependencies [Windows]
+      if: matrix.os == 'windows-latest'
+      run: |
+        ${env:VCPKG_INSTALLATION_ROOT}/vcpkg.exe install --triplet x64-windows opencv3[contrib] vtk hdf5 fftw3
```
Comment (pattacini) [inline .github/workflows/ci.yml]: still failing...
any suggestion @traversaro ?

**Pull request merged**